src/components/shared/card.tsx

```diff
@@ -0,0 +1,166 @@
+import { Pillar, Design, Display } from '@guardian/types/Format';
+import { Image } from 'image';
+import { darkModeCss } from 'styles';
+import { RelatedItemType } from '@guardian/apps-rendering-api-models/relatedItemType';
+import { Pillar as contentPillar } from '@guardian/content-api-models/v1/pillar';
+import { getPillarStyles, pillarFromString } from 'pillarStyles';
+
+
+
+interface Props {
+    relatedItem: RelatedItem;
+    image: Option<Image>;
+}
+
+const listStyles = css`
+    background: white;
+    margin-right: ${remSpace[3]};
+    flex: 0 0 15rem;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    img {
+        width: 100%;
+        height: 100%;
+        position: absolute;
+        top: 0;
+        left: 0;
+    }
+
+    time {
+        ${textSans.small()};
+        color: ${text.supporting};
+        text-align: right;
+        width: calc(100% - ${remSpace[2]});
+        display: inline-block;
+    }
+`
+
+const anchorStyles = css`
+    color: ${neutral[7]};
+    text-decoration: none;
+    ${darkModeCss`
+        color: ${neutral[86]};
+        background: ${neutral[20]};
+    `}
+`;
+
+const headingStyles = css`
+    ${headline.xxxsmall()};
+    margin: 0;
+    padding: ${remSpace[2]};
+    min-height: 150px;
+`;
+
+const imageWrapperStyles = css`
+    padding-bottom: 56.25%;
+    position: relative;
+`;
+
+const relativeFirstPublished = (date: Option<Date>): JSX.Element | null => pipe2(
+    date,
+    map(date => <time>{makeRelativeDate(date)}</time>),
+    withDefault<JSX.Element | null>(null),
+);
+
+const cardStyles = (type: RelatedItemType, pillar: contentPillar): SerializedStyles => {
+    switch(type) {
+        case RelatedItemType.ARTICLE: {
+            return css``;
+        }
+
+        case RelatedItemType.FEATURE: {
+            const { kicker } = getPillarStyles(pillarFromString(pillar.id))
+
+            return css`
+                h2 {
+                    ${headline.xxxsmall({ fontWeight: 'bold' })}
+                    color: ${kicker};
+                }
+            `;
+        }
+
+        case RelatedItemType.ANALYSIS: {
+            return css``;
+        }
+
+        case RelatedItemType.GALLERY: {
+            return css``;
+        }
+
+        case RelatedItemType.SPECIAL: {
+            return css``;
+        }
+
+        case RelatedItemType.AUDIO: {
+            return css``;
+        }
+
+        case RelatedItemType.LIVE: {
+            return css``;
+        }
+
+        case RelatedItemType.VIDEO: {
+            return css``;
+        }
+
+        case RelatedItemType.REVIEW: {
+            return css``;
+        }
+
+        case RelatedItemType.ADVERTISEMENT_FEATURE: {
+            return css``;
+        }
+
+        case RelatedItemType.COMMENT: {
+            return css``;
+        }
+    }
+
+    return css``;
+}
+
+const Card = ({ relatedItem, image }: Props): JSX.Element => {
+
+    const format = {
```

Comment:
Should this be a `prop` passed to `Card`?

Reply:
Maybe this can be done in a following PR, when we generate a format from the data we get from MAPI